web/jest.config.ts

```diff
@@ -8,6 +8,8 @@ export default async (): Promise<Config.InitialOptions> => {
     setupFilesAfterEnv: ["./setupTests.js"],
     testEnvironment: "jsdom",
     testPathIgnorePatterns: ["<rootDir>/.next/", "<rootDir>/node_modules/", "<rootDir>/cypress/"],
+    rootDir: "./",
+    moduleDirectories: ["node_modules", "<rootDir>"],
```

Comment:
If these two additions are necessary, I'd separate them out into a separate chore.

Reply:
I imagine you mean separate commits not separate pivotal tickets?

This is necessary, because if we're using the seo.config.js which is the the root directory our jest tests can't actually see that file unless the root directory is a moduleDirectory

the rootDir declaration is just a good habit and the node_modules are necessary because I believe it implicitly grabs them unless you declare this moduleDirectories field in which case you have to be explicit.